<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1550498355756" stop="1550498355799">
  <name>scripts</name>
  <labels/>
  <test-cases>
    <test-case start="1550498355756" status="passed" stop="1550498355758">
      <name>test_login.TestLogin.test_login1</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="28825-MainThread"/>
        <label name="host" value="itheima.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1550498355759" status="passed" stop="1550498355760">
      <name>test_login.TestLogin.test_login2</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="28825-MainThread"/>
        <label name="host" value="itheima.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1550498355760" status="failed" stop="1550498355799">
      <name>test_login.TestLogin.test_login3</name>
      <failure>
        <message>AssertionError: assert 0</message>
        <stack-trace>self = &lt;scripts.test_login.TestLogin object at 0x1073cd198&gt;

    def test_login3(self):
&gt;       assert 0
E       assert 0

scripts/test_login.py:10: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="28825-MainThread"/>
        <label name="host" value="itheima.local"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
